.github/workflows/build-repo.yml

```diff
@@ -6,9 +6,6 @@ on:
 jobs:
   build-test-release:
     runs-on: ubuntu-latest
-    strategy:
```

Comment:
With this we lose the ability to specify a future version for testing/building before we try to move to it.

Worth the tradeoff to use the `.nvmrc` directly here? It is really nice to not duplicate the 'current' version in the matrix.

Reply:
More up to you guys. It doesn't effect my workflow (local) either way. 
If it's a change you'd like reverted I a-okay with that.

Reply:
@mshriver

Reply:
For now we'll leave it as is, if you wish to change it in the future we can do that quite easily.

Reply:
Missed your response - not taking a hard stance here, just thought it was worth consideration. I doubt we'll be moving node versions enough to make it necessary.

Agreed, if we find ourselves wanting next version results early we can modify without much trouble.